### DISCUSSION TITLE
macOS-arm64 v1.0.9 can't be print result.

### DISCUSSION THREAD

**Discussion by dollce** [projectdiscovery/dnsx#196] [General]
### dnsx version:

`v1.0.9`

### Current Behavior:

Doesn't print anything.

### Expected Behavior:

Something print by result.

### Steps To Reproduce:

1. Run `dnsx -l dns.txt -resp -a -aaaa -cname`
> dns.txt have values.
> and `dnsx -ptr -resp-only` command also doesn't work.
2. printed signature and warning caption
3. Nothing happened and exited

### Anything else:

![image](https://user-images.githubusercontent.com/73532293/158012489-56948cd9-b9e1-45da-8abd-48a7c11c1116.png)

I also installed with homebrew and github release and git clone build.
but all doesn't work.

**Comment by ehsandeep** [comment]
@brucejhju checked with `v` flag to see what's going on? also any example to share from `dns.txt`?

**Comment by dollce** [comment]
> @brucejhju checked with `v` flag to see what's going on? also any example to share from `dns.txt`?

@ehsandeep 

- `-v` option also doesn't work.

- `dns.txt`

same as sample url in README

**Comment by ehsandeep** [comment]
Unable to replicate this
